qiskit_machine_learning/algorithms/classifiers/pegasos_qsvc.py

```diff
@@ -84,11 +84,13 @@ def __init__(
 
         Raises:
             ValueError:
-                - if ``quantum_kernel`` is passed and ``precomputed`` is set to ``True``. To use
-                a precomputed kernel, ``quantum_kernel`` has to be of the ``None`` type.
+                - if :class:`~qiskit_machine_learning.kernels.FidelityQuantumKernel` is passed and
+                ``precomputed`` is set to ``True``. To use a precomputed kernel,
+                 :class:`~qiskit_machine_learning.kernels.FidelityQuantumKernel` has to be of
+                 the ``None`` type.
             TypeError:
-                - if ``quantum_kernel`` neither instance of
-                  :class:`~qiskit_machine_learning.kernels.BaseKernel` nor ``None``.
+                - if :class:`~qiskit_machine_learning.kernels.FidelityQuantumKernel` neither instance
+                  of :class:`~qiskit_machine_learning.kernels.BaseKernel` nor ``None``.
```

Comment:
This I think can be completely removed - the code did raise a TypeError but that seems to have been removed in a recent change since things are now more general around  a BaseKernel rather than what was there before.


On thing I do note is that is that this routine raises a ValueError if C <= 0 i.e. not a positive number, which is not included above.